src/query.rs

```diff
@@ -109,6 +174,7 @@ impl<S> QueryRef<'_, S>
 where
     S: QuerySpec,
 {
+    /// Create an iterator over the query's matches.
```

Comment:
I think the short possessive form is usually restricted to people. Maybe "Create an iterator over the entities matching the query."?